### DISCUSSION TITLE
Light Puzzles

### DISCUSSION THREAD

**Discussion by mgatelabs** [mgatelabs/GenshinSolvers#3] [Ideas]
They are possible, i was thinking and the only difference is instead of directions, it is number of lights.

Light puzzles are now in and functional.  And technically light and spinning puzzles are the same, they each have four states, just one lights up and the other spins.

**Comment by stefnotch** [comment]
I wonder, can discussions be closed or archived? This one doesn't seem to be very relevant anymore, now that we've gotten both working

**Reply by mgatelabs** [reply]
I can nuke them or lock them.

**Reply by stefnotch** [reply]
I guess let's try locking it?
